### ISSUE TITLE
Add a CLI to create objects supported by SecretGen controller

### ISSUE THREAD

**Issue by Zerpet** [carvel-dev/secretgen-controller#540]
**Describe the problem/challenge you have**

YAML is hard to remember. Every time I want to create an object of this controller, I have to look at something I've done before, or come here to read the docs.

**Describe the solution you'd like**

A CLI, similar to `kctrl` (much Kudos for `kctrl` 👏 ), that allows to create some, or all, of the objects supported by this controller. As an example, I would like to add a `SecretExport` by doing:

```console
secgenctl secretexport create my-secret-export -n my-super-secret-namespace --to-namespaces '*'
```

This is an example, the idea is to support create/list/get/delete operations using this CLI for all objects.

**Anything else you would like to add:**

To my mind is a lot easier to remember CLI commands than YAML structures. A CLI can provide a `--help` option, and that's faster than opening a browser and navigating to the docs of this controller.

I'll be happy to contribute a PR!

---
Vote on this request

This is an invitation to the community to vote on issues, to help us prioritize our backlog. Use the "smiley face" up to the right of this comment to vote.

👍 "I would like to see this addressed as soon as possible"
👎 "There are other more important things to focus on right now"

We are also happy to receive and review Pull Requests if you would like to work on this issue.

**Comment by 100mik** [comment]
We would definitely prioritise reviews and help you with the process if you are willing to work on this!
We generally start larger efforts like these with a proposal, I would be happy to help you get started on it 🙌🏼

**Comment by Zerpet** [comment]
Yeah, sounds good! I'll be doing this during my free time, bear with me if I take some time to respond from time to time 🙂 How do I get started with the proposal process?
